codespell_lib/data/dictionary.txt

```diff
@@ -8078,6 +8078,7 @@ failes->fails
 faill->fail
 failled->failed
 failre->failure
+failsave->failsafe
```

Comment:
Wikipedia seems conflicted, but fail-safe looks to be the most used version, so probably suggest that first, and then maybe the non-hyphenated one.

Reply:
be1b33e